src/coreclr/jit/targetloongarch64.h

```diff
@@ -0,0 +1,332 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+#pragma once
+
+#if !defined(TARGET_LOONGARCH64)
+#error The file should not be included for this platform.
+#endif
+
+// NOTE for LoongArch64:
+//  The `REG_R21` which alias `REG_X0` is specially reserved !!!
+//  It can be used only by manully and should be very careful!!!
+
+// clang-format off
+  #define CPU_LOAD_STORE_ARCH      1
+  #define CPU_HAS_FP_SUPPORT       1
+  #define ROUND_FLOAT              0       // Do not round intermed float expression results
+  #define CPU_HAS_BYTE_REGS        0
+
+  #define CPBLK_UNROLL_LIMIT       64      // Upper bound to let the code generator to loop unroll CpBlk.
+  #define INITBLK_UNROLL_LIMIT     64      // Upper bound to let the code generator to loop unroll InitBlk.
+
+#ifdef FEATURE_SIMD
+#pragma error("SIMD Unimplemented yet LOONGARCH")
+  #define ALIGN_SIMD_TYPES         1       // whether SIMD type locals are to be aligned
+  #define FEATURE_PARTIAL_SIMD_CALLEE_SAVE 1 // Whether SIMD registers are partially saved at calls
+#endif // FEATURE_SIMD
+
+  #define FEATURE_FIXED_OUT_ARGS   1       // Preallocate the outgoing arg area in the prolog
+  #define FEATURE_STRUCTPROMOTE    0       // JIT Optimization to promote fields of structs into registers
```

Comment:
You should probably open GitHub issues to track "TODO" items, such as enabling struct promotion. Of course, you can decide how you want to track these things.

Reply:
Ok, Thanks very much.
These features should be added by individual PR.